photon-ml/src/main/scala/com/linkedin/photon/ml/algorithm/CoordinateDescent.scala

```diff
@@ -40,7 +42,7 @@ class CoordinateDescent(
     coordinates: Seq[(String, Coordinate[_ <: DataSet[_], _ <: Coordinate[_, _]])],
     trainingLossFunctionEvaluator: Evaluator,
     validatingDataAndEvaluatorsOption: Option[(RDD[(Long, GameDatum)], Seq[Evaluator])],
-    logger: PhotonLogger) {
+    logger: Logger) {
```

Comment:
PhotonLogger is an implementation of Logger. These classes shouldn't care about the backing implementation.